ecommerce/clothing_catalogue/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('products/', views.product_list, name='product-list'),
```

Comment:
```suggestion
    path('products/', views.product_list, name='product_list'),
```

urls have dashes `-` and URL names have underscores